src/ui/public/doc_table/components/table_row/_open.scss

```diff
@@ -0,0 +1,14 @@
+/**
+ * 1. When switching between open and closed, the toggle changes size
+ *    slightly which is a problem because it forces the entire table to
+ *    re-render which is SLOW.
```

Comment:
Can the comment be clear if the SLOW what happens with/from this code, or if it is what this code fixes?

Reply:
This was moved from the discover styles, I kept the comment with it, but I didn't write the comment, so I'm actually not sure... haha

https://github.com/elastic/kibana/blob/eb43306a1b8d6167521d66e6ab75c86c3706733d/src/core_plugins/kibana/public/discover/_discover.scss#L71-L83